server/statistics/hot_peer.go

```diff
@@ -140,10 +137,9 @@ func (stat *HotPeerStat) Log(str string, level func(msg string, fields ...zap.Fi
 		zap.Int("hot-degree", stat.HotDegree),
 		zap.Int("hot-anti-count", stat.AntiCount),
 		zap.Duration("sum-interval", stat.getIntervalSum()),
-		zap.Bool("need-delete", stat.IsNeedDelete()),
 		zap.String("source", stat.source.String()),
 		zap.Bool("allow-adopt", stat.allowAdopt),
-		zap.Bool("just-transfer-leader", stat.justTransferLeader),
+		zap.String("operation-type", stat.opType.String()),
 		zap.Time("last-transfer-leader-time", stat.lastTransferLeaderTime))
```

Comment:
The in-cold should be added into the log.

Reply:
Since the original log doesn't contain this field, I'm going to keep it as it is. Besides, this log is too complicated even if it is a debug log.